Makefile

```diff
@@ -136,13 +137,12 @@ $(PROMU):
 	GOOS= GOARCH= go get -u github.com/prometheus/promu
 
 # Always pin dep to the correct version. It changes too often.
-# TODO(bplotka): Install dep without using user's `github.com/golang/dep` package. (Use some tmp place).
 $(DEP):
 	@echo ">> fetching dep from 45be32ba4708aad5e2aa8c86f9432c4c4c1f8da2 revision"
-	@go get -d -u github.com/golang/dep/cmd/dep
-	@cd $(FIRST_GOPATH)/src/github.com/golang/dep && git checkout -f -q 45be32ba4708aad5e2aa8c86f9432c4c4c1f8da2
-	@go install github.com/golang/dep/cmd/dep
-	@mv $(FIRST_GOPATH)/bin/dep $(DEP)
+	@GOPATH=$(DEP_GOPATH) go get -d -u github.com/golang/dep/cmd/dep
```

Comment:
Can we ensure that this directory is empty? Try to remove it first? Otherwise, if we change versions and someone already installed dep using this, we might have git conflict anyway.

Reply:
this directory = `/tmp/thanos-go`